OneSignalSDK/onesignal/src/main/java/com/onesignal/OSInAppMessage.java

```diff
@@ -46,4 +49,16 @@ public String getMessageId() {
         return messageId;
     }
 
+    public JSONObject toJSONObject() {
+        JSONObject mainObj = new JSONObject();
+        try {
+            mainObj.put("messageId", messageId);
```

Comment:
I think that in other new classes, all strings are put inside a constant inside the file. Let me know if you agree with putting "messageId" inside a private static final String

Reply:
ah good catch, I agree with that

Reply:
Ah ok! I have seen that practice but didn't realize it is preferred.